Automattic/woocommerce-payments
# Remove image preview for Disputes Evidence upload fields settings

Fixes #3853

#### Changes proposed in this Pull Request
Disputes evidence upload field settings to display file name instead of preview as the uploaded files are not only images.

#### Testing instructions
- Go to 'Payments' -> 'Disputes'.
- Open one of the disputes and click Challenge dispute.
- Select 'Physical product' from the `Product type` dropdown.
- Upload a file to the `Custom signature` setting field.
- Confirm the preview of the file is not displayed and the file name is displayed instead.
--------------------
